python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -577,7 +667,7 @@ def __init__(cls, name, bases, this_dict):
 
 
 class GangaObject(Node):
-    __metaclass__ = ObjectMetaclass
+    __metaclass__ = ObjectMetaclass # This overloads the default metaclass so we can do some clever stuff
```

Comment:
yes but it was undocumented and I wanted to remind myself that the `__metaclass__` is defined within the python object not something inherited and constructed in Node or somewhere else.